cypress/scripts/run-as-package.js

```diff
@@ -5,6 +5,8 @@ const path = require('path')
 const utils = require('../../__tests__/spec/utils')
 
 const testDir = path.resolve(process.env.KIT_TEST_DIR || 'cypress/temp/test-project')
+fs.rmdirSync(testDir, { recursive: true })
```

Comment:
I'd rather we didn't do this... it makes tests slower for me 😅

Reply:
Removed